decoder/src/log.rs

```diff
@@ -75,8 +75,7 @@ impl<'a> DefmtRecord<'a> {
         }
 
         let timestamp = &target[DEFMT_TARGET_MARKER.len()..];
-
-        Some(Self {
+        (Self{
```

Comment:
does this compile? the return type is an option but this no longer has the `Some` constructor